code-challenges.js

```diff
@@ -21,29 +21,133 @@ const temp1 = 42
 const temp2 = 350
 const temp3 = 212
 
+// Pseudo Coding 
+// Create a function that takes any given number 
+// within the function determines if the number is below boiling, at boiling point or above boiling point
+// boiling point is 212 degrees Farenheit
+//I should expect 42 is below boiling point, 350 is above boiling point & 212 is at boiling point
+//within the function i would write and else if statement that determine is below boiling point is 42, else if above pointing is 350 and else if 212 is at boiling.
+// when I tried every number it gave the correct response
+// I received help with this question/challenge
+
+const boilingPoint = (temp) => {
+if(temp == 212){
+    return "${temp} is below boiling Point"
+} else if (temp>212){
+    return "${temp} is above the boiling point"
+}else if (temp< 212) {
+    return "${temp} is below the boiling point"
```

Comment:
Your logic looks great and your pseudocode is outstanding!  When using string interpolation, just make sure that you use the backticks `` instead of quotes.  This allows javascript to inject the variables into the string.  Also, just watch your indentation.